packages/amplify-graphql-transformer-core/src/transform-host.ts

```diff
@@ -1,18 +1,16 @@
 import { TransformHostProvider } from '@aws-amplify/graphql-transformer-interfaces';
```

Comment:
It looks like there are some left over uses in comments in this file.

Reply:
fixed 👍🏼